src/components/ResponsiveAppBar.jsx

```diff
@@ -27,7 +27,7 @@ const pages = [
   { url: "eco-score", translationKey: "menu.eco-score" },
   { translationKey: "menu.manage" },
   { url: "insights", translationKey: "menu.insights" },
-  // { url: "settings", translationKey: "menu.settings" },
```

Comment:
think removing this element will affect the `{displayedPages.map((page) =>` below